### ISSUE TITLE
[Suggestion] Elevator block emits a redstone signal

### ISSUE THREAD

**Issue by ExDomino** [McJty/RFTools#1307]
Hello, my suggestion is simple: the elevator block could emits a redstone signal on its top when the plateform reach its level or when the call button is pressed and the elevator is already on this floor. The delay could also be specified inside the elevator block GUI. It would be really useful for door that are automated open and closed by the platform itself. For now we need the ExtraUtilities scanner to do that and MalisisDoor to create true elevator doors that open on each sides.

**Comment by ExDomino** [comment]
But it's not on top of it, so doors won't open.

**Comment by McJty** [comment]
Well that can be fixed with some redstone logic can't it?

**Comment by ExDomino** [comment]
No, because you don't have place for that. If you want to make that compact enough to make that nobody notice the redstone logic except the Elevator block itself, you cannot place any logic redstone there. The Scanner from Extra Utilities can detect the platfom and emit above of it to open the door, but behind the Elevator block, if it's under the door, it's the floor.
